src/Form/Extension/AceEditor/Type/AceEditorType.php

```diff
@@ -45,62 +54,71 @@ public function configureOptions(OptionsResolver $resolver): void
 
                 return $value;
             }
-            if (preg_match('/([0-9\.]+)\s*('.implode('|', self::UNITS).')/', (string) $value, $matchedValue)) {
+            if (preg_match('/([0-9\.]+)\s*(' . implode('|', self::UNITS) . ')/', (string)$value, $matchedValue)) {
                 $value = $matchedValue[1];
-                $unit = $matchedValue[2];
+                $unit  = $matchedValue[2];
             } else {
                 $unit = self::DEFAULT_UNIT;
             }
 
             return ['value' => $value, 'unit' => $unit];
         };
 
+        $useStimulus = false;
+        if (in_array(StimulusBundle::class, $this->installedBundles, true) && interface_exists(AssetMapperInterface::class)) {
+            $useStimulus = true;
+        }
```

Comment:
I think I would extract this logic to boolean parameter that default value would be dynamically set by container compiler pass. Container compiler pass should however respect this value if set by user in bundle configuration manually